src/app/core/assessment.model.ts

```diff
@@ -11,4 +11,5 @@ export class Assessment {
   public garage: string;
   public latitude: number;
   public longitude: number;
+  public assessed_value: number;
```

Comment:
Is it safe to remove the `total_asmt` on line 6?

Reply:
Yes it should be safe to remove total_asmt from assessment.model.ts.